apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="example">
+  <div class="header-example-container">
+    <p>
+      Header is a component that manages the layout of the header section of kirby-page. It is
+      responsive and will change the layout according to the device size. It allows for a wide range
+      of use cases including support for avatar, flags etc.
+    </p>
+
+    <h2>Default</h2>
+
+    <cookbook-example-viewer [html]="defaultExample.template">
+      <div class="header-example">
+        <cookbook-header-example-default #defaultExample></cookbook-header-example-default>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Flag</h2>
+    <p>
+      A flag can be used to indicate a status. To use flag you simply add
+      <code>kirby-flag</code>
+      in the header and that should position the flag correctly and manage spacing.
```

Comment:
Consider wording:
```suggestion
      in the header which will position the flag correctly and manage spacing.
```

Reply:
@thupi please consider changing this (and align it with the explaining text for avatar)